uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/leave-request-ctrl_test.js

```diff
@@ -484,7 +484,7 @@
             });
 
             it('resets balance and types', function () {
-              expect($ctrl.balance.closing).toEqual(0);
+              expect($ctrl.balance.closing).toEqual($ctrl.balance.opening);
```

Comment:
Please add a comment in the code explaining why `$ctrl.balance.closing` would be same as `$ctrl.balance.opening`